probe/endpoint/ebpf_linux.go

```diff
@@ -0,0 +1,7 @@
+//+build linux
```

Comment:
I don't think this is needed (the filename determines it should be only built in linux)